InertiaCore/Utils/Vite.cs

```diff
@@ -0,0 +1,147 @@
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace InertiaCore.Utils;
+
+public static class Vite
+{
+
+    private static string GetString(IHtmlContent content)
+    {
+        var writer = new System.IO.StringWriter();
+        content.WriteTo(writer, HtmlEncoder.Default);
+        return writer.ToString();
+    }
+
+    private static string publicDir(string path)
```

Comment:
Use `protected` instead of `private` in every private method here.

Reply:
@kapi2289 Apparently only `private` and `public` are allowed on static classes in C#, since they cannot be inherited by other classes. I opted to make most things private, except if we need to expose functionality for interacting with the class.